packages/fxa-admin-panel/package.json

```diff
@@ -61,52 +59,36 @@
   "devDependencies": {
     "@babel/core": "^7.15.0",
     "@rescripts/cli": "0.0.16",
-    "@storybook/addon-actions": "^6.4.12",
-    "@storybook/addon-links": "^6.3.13",
-    "@storybook/addons": "^6.3.12",
```

Comment:
These might be used in the storybook config file: https://github.com/mozilla/fxa/blob/6258ae5f6f574f36c388002e672a6099202d5ca6/packages/fxa-admin-panel/.storybook/main.js#L5-L8

Although,  might be a bit moot since I don't think that fxa-admin-panel has any stories: https://storage.googleapis.com/mozilla-storybooks-fxa/commits/9bead54a9e22d344dd7772bfbce029586cc281b4/fxa-admin-panel/index.html

Reply:
Yes, we have a ticket somewhere for this. Storybook was added, but we never wrote any stories. I am not sure what to do here. I think we can just remove it until stories are actually created.

Reply:
Yes. If it's not in use let's get rid of it